### PR TITLE
random_uniform

### PR DESCRIPTION
**Pull request by rnwzd**
#1256

### CODE REVIEW

File: ivy/functional/ivy/random.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def random_uniform(
         If the given shape is, e.g ``(m, n, k)``, then ``m * n * k`` samples are drawn.
         If size is ``None`` (Default), a single value is returned.
     device
-        device on which to create the array 'cuda:0', 'cuda:1', 'cpu' etc.
+        device on which to create the array 'gpu:0', 'gpu:1', 'cpu' etc.
```
Comment (chie2727) [inline ivy/functional/ivy/random.py:38]: You can change this part back to `'cuda:0', 'cuda:1', 'cpu' etc. `

Also in a new line, please include: ` (Default value = None)` just like in Line 226.

**Review by chie2727 — CHANGES_REQUESTED** [review]

**Review by chie2727 — CHANGES_REQUESTED** [review]
Hi @rnwzd ,
Please also include an Instance Method example for ivy.Container instances.

For example:
https://github.com/unifyai/ivy/blob/2045db570d7977830681a7498a3c1045fb5bcc79/ivy/functional/ivy/elementwise.py#L234

Also please leave 1 line of space between each example. You also need to organize them under the `Functional Examples` and `Instance Method Examples` headings.

**Comment by rnwzd** [comment]
> 

 I am sorry but I don't see which argument accepts container instances.

**Comment by chie2727** [comment]
You can try placing `ivy.random_uniform()` into a container and generate an example from this

**Comment by chie2727** [comment]
Also, for your reference have a look at the Docstring Examples guide to get an idea of the overall layout
https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html

File: ivy/functional/ivy/random.py [hunk 2]
```diff
@@ -51,18 +51,42 @@ def random_uniform(
...
-    Examples
-    --------
-    >>> y = ivy.random_uniform(0.0, 2.0)
+    Functional Examples
+    -------------------
+    
+    >>> y = ivy.random_uniform()
     >>> print(y)
-    ivy.array(1.89150229)
+    ivy.array(0.26431865)
```
Comment (chie2727) [inline ivy/functional/ivy/random.py:67]: Thanks for all the edits!

As this function has 4 input parameters, you need a minimum of 4 types of functional example variations.
So could you include 1 more example that takes maybe 2 parameters.
After that you should be done!

**Review by chie2727 — CHANGES_REQUESTED** [review]

**Comment by chie2727** [comment]
Thanks for all the edits!

As this function has 4 input parameters, you need a minimum of 4 types of functional example variations.
So could you include 1 more example that takes maybe 2 parameters.

After that you should be done!

**Review by chie2727 — CHANGES_REQUESTED** [review]
.

**Review by rnwzd — COMMENTED** [review]
ok.
SHould be fixed now.
At the moment the button to request a review is not working though.

**Review by chie2727 — APPROVED** [review]

**Comment by chie2727** [comment]
Thanks for adding in the last example. I'll merge in your changes now.

**Pull request merged**